cypress/e2e/Form.cy.ts

```diff
@@ -85,6 +85,9 @@ describe('App', () => {
     cy.get('[data-cy="Pipeline version-categorical-field"] input').should('not.be.disabled');
     cy.get('[data-cy="Pipeline version-categorical-field"]').type('0.2.3{downarrow}{enter}');
     cy.get('[data-cy="Pipeline version-categorical-field"] input').should('have.value', '0.2.3');
+    cy.get('[data-cy="Pipeline name-categorical-field"]').clear();
+    cy.get('[data-cy="Pipeline version-categorical-field"] input').should('be.disabled');
+    cy.get('[data-cy="Pipeline version-categorical-field"]').should('have.value', '');
```

Comment:
this works, but I think it would be better as a separate test to keep it one test, one problem.

Reply:
like a separate test case i.e., `it` or a different test file?

Reply:
I'd say `it` is completely enough